pkg/admissioncontroller/webhooks/auth/seed/authorizer.go

```diff
@@ -150,6 +164,20 @@ func (a *authorizer) authorizeEvents(seedName string, attrs auth.Attributes) (au
 	return auth.DecisionAllow, "", nil
 }
 
+func (a *authorizer) authorizeLease(seedName string, attrs auth.Attributes) (auth.Decision, string, error) {
+	if attrs.GetName() == "gardenlet-leader-election" &&
+		utils.ValueExists(attrs.GetVerb(), []string{"create", "get", "watch", "update"}) {
+
+		return auth.DecisionAllow, "", nil
+	}
```

Comment:
Do I get it correctly that the use case for this is when you don't have a virtual Garden, i.e., seed == garden? Won't this request subsequently be denied by the SeedRestriction admission?

Reply:
Indeed, thank you very much for catching this! Fixed with 9c74d19fd